portal-impl/test/unit/com/liferay/portal/util/PortalImplEscapeRedirectTest.java

```diff
@@ -59,6 +64,18 @@ public void setUp() {
 		_originalRedirectURLSettings = ReflectionTestUtil.getAndSetFieldValue(
 			RedirectURLSettingsUtil.class, "_redirectURLSettings",
 			_redirectURLSettingsImpl);
+
```

Comment:
For the failing test in this file, I think the main point of the test is to check the behavior of _portalImpl.escapeRedirect() while securityMode is "domain" and "id". 
Therefore, even though it is not a validPortalDomain, it will not affect the test, and we just need to make sure NPE is not thrown